test/test_linear_operators.jl

```diff
@@ -150,6 +206,65 @@ y2 = ifft(x1)
 
 @test all(norm.(y1 .- y2) .<= 1e-12)
 
+n = 4
+op = IDFT(Float64,(n,),1)
+x1 = randn(n)
+y1 = test_op(op, x1, fft(randn(n)), verb)
+y2 = ifft(x1,1)
+
+@test all(norm.(y1 .- y2) .<= 1e-12)
+
+op = IDFT(Complex{Float64},(n,),1)
+x1 = randn(n)+im*randn(n)
+y1 = test_op(op, x1, fft(randn(n)), verb)
+y2 = ifft(x1,1)
+
+@test all(norm.(y1 .- y2) .<= 1e-12)
+
+op = AbstractOperators.IDFT(Float64,(n,n))
```

Comment:
perhaps here something like `(n,m)` where `m != n` would make the test more robust.

Reply:
Good point, will add some more robust tests.